CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ if(CMAKE_C_COMPILER_ID MATCHES "^(Intel)$")
 endif()
 
 # Find packages.
-find_package(NetCDF 4.7.3 REQUIRED C Fortran)
+find_package(NetCDF 4.3.3 REQUIRED C Fortran)
```

Comment:
Actually I don't know that we should do this. We are not testing with 4.3.3. We really don't know that this code works with that version, because we don't have much testing anyway.

We have best confidence that it works with 4.7.4. I think we should require that.